### DISCUSSION TITLE
New Taiko Control Scheme for Android?

### DISCUSSION THREAD

**Discussion by reissfelt** [ppy/osu#30149] [Ideas]
Hello, i've been playing Taiko with relatively weird layouts of DKKD. I've playing like this for so many years since i started touching Taiko on osu!. Recently i played the Android version to find out it's so difficult for me to play on Android since i have to cross my hand in order to play, yet it's still hard to play with that.

I'm wondering if you could make a DKKD Touch Control Scheme for that. Thanks in advance.

**Comment by yosh890** [comment]
I think it should be made that taiko on mobile can support any playstyle you want... DDKK is a common playstyle (and apparently easier for beginners) and does not appear to be supported. Would it be at all possible for different playstyles of taiko to be supported on mobile like on desktop?

**Reply by reissfelt** [reply]
I would love to if this was implemented
